middleware/auth.ts

```diff
@@ -6,4 +6,10 @@ export default defineNuxtMiddleware((context) => {
    * @todo Redirect the user to main page if token is present in store
    * @hints check what propeties context has
    */
+  
+
+  if(!context.store.getters.auth)
+  {
+   return context.redirect('/login/');
```

Comment:
you should have made sure that if user is logged in he shouldn't be able to go on /login